docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -269,6 +307,24 @@ Once you have added a query to your template's front matter block, execution hap
 {{/each}}
 {{/if}}
 ```
+
+<!--
+type: tab
+title: Category
+-->
+
+```handlebars title="Handlebar in category.html template" lineNumbers

```

Comment:
```suggestion
```handlebars title="Example gql Handlebars object in category.html template" lineNumbers

```